cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOpenInvoiceData.ds

```diff
@@ -36,14 +36,14 @@ importPackage( dw.web );
 var AdyenHelper = require ("int_adyen_overlay/cartridge/scripts/util/AdyenHelper");
 
 function getShopperData(args, personalDetails){
-	if(args.socialSecurityNumber){
-		personalDetails.socialSecurityNumber = args.socialSecurityNumber;
+	if(args.adyenForm.socialSecurityNumber.value){
```

Comment:
if(args.adyenForm.socialSecurityNumber && args.adyenForm.socialSecurityNumber.value != null)

Reply:
Not applicable, keeping current code